Файл: tests/__pycache__/__init__.cpython-312.pyc
Серьезность: low
Сообщение: Sample issue found in unknown file type
Строка: 10

Файл: tests/__pycache__/test_calculator.cpython-312-pytest-7.4.4.pyc
Серьезность: low
Сообщение: Sample issue found in unknown file type
Строка: 10

Файл: tests/__pycache__/test_sample.cpython-312-pytest-7.4.4.pyc
Серьезность: low
Сообщение: Sample issue found in unknown file type
Строка: 10

Файл: tests/fail.py
Серьезность: high
Сообщение: Potential issue: usage of print() found in Python
Строка: 9

Файл: tests/test.py
Серьезность: high
Сообщение: Potential issue: usage of print() found in Python
Строка: 14

